types/complex/index.d.ts

```diff
@@ -61,19 +72,25 @@ export default class Complex {
     fromPolar(r: number, phi: number): Complex;
 
     /**
-     * Set the precision of the numbers. Similar to Number.prototype.toPrecision. Useful before printing the number with the toString method.
+     * Set the precision of the numbers. Similar to Number.prototype.toPrecision.
+     * Useful before printing the number with the toString method.
      * @param k An integer specifying the number of significant digits
      */
     toPrecision(k: number): Complex;
 
     /**
-     * Format a number using fixed-point notation. Similar to Number.prototype.toFixed. Useful before printing the number with the toString method.
-     * @param k The number of digits to appear after the decimal point; this may be a value between 0 and 20, inclusive, and implementations may optionally support a larger range of values. If this argument is omitted, it is treated as 0.
+     * Format a number using fixed-point notation. Similar to Number.prototype.toFixed.
+     * Useful before printing the number with the toString method.
+     * @param k The number of digits to appear after the decimal point [0 - 20].
+     * Implementations may optionally support a larger range of values.
+     * If this argument is omitted, it is treated as 0.
```

Comment:
Does this mean the parameter should have a default value?

Reply:
Thanks, will investigate